sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -551,13 +716,25 @@ public ServiceCall commitBlockListWithRestResponse(String containerName,
      * Builder for {@link StorageBlobClient}.
      */
     public static class Builder {
+        private final String storageBlobClientId;
         private final ServiceClient.Builder serviceClientBuilder;
+        private final Constraints.Builder transferConstraintsBuilder;
+        private static final StorageBlobClientMap STORAGE_BLOB_CLIENTS;
+
+        static {
+            STORAGE_BLOB_CLIENTS = StorageBlobClientMap.getInstance();
+        }
 
         /**
          * Creates a {@link Builder}.
+         *
+         * @param storageBlobClientId the unique id for the {@link StorageBlobClient} this builder builds.
+         *     This identifier is used to associate this {@link StorageBlobClient} with the upload, download transfers
+         *     it initiates. When a transfer is reloaded from disk (e.g. after an application crash), it can only be
+         *     resumed once a client with the same storageBlobClientId has been initialized.
```

Comment:
```suggestion
         * @param storageBlobClientId The unique ID for the {@link StorageBlobClient} this builder builds. This
         *                            identifier is used to associate this {@link StorageBlobClient} with the upload and
         *                            download transfers it initiates. When a transfer is reloaded from disk (e.g.
         *                            after an application crash), it can only be resumed once a client with the same
         *                            storageBlobClientId has been initialized.
```